.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,36 @@ Optionally add one or more of the following kinds if applicable:
 /kind regression
 -->
 
+**Which SIG(s) this PR belongs to?**
+
+<!--
+Add one or more of the following SIGs:
```

Comment:
Please add a link to the contributor documentation that explains SIGs and how to find the proper SIG!

Reply:
Done.